build_html.py

```diff
@@ -13,6 +13,7 @@
 when_to_start = int(os.environ.get('WHEN_TO_START')) # first hour in output table
 hours_to_print = int(os.environ.get('HOURS_TO_PRINT')) # how many hours worth of data in table
 outfile = str(os.environ.get('OUTFILE'))
+use_keyword_lists = int(os.environ.get('USE_KEYWORD_LIST'))
```

Comment:
This looks great, but use_keyword_lists is also defined later in this file, around line 35. I think we just have to comment out that second variable assignment.